README.md

```diff
@@ -59,6 +59,16 @@ zmqpubrawblock=tcp://127.0.0.1:29000
 zmqpubrawtx=tcp://127.0.0.1:29000
 ```
 
+Depending on the actual hardware configuration, it may be useful to provide increased `dbcache` parameter value for faster verification and `rpcthreads` parameter value for better handling of multiple simultaneous API requests from `bitcoind` side. 
+
+```conf
+dbcache=2048
+rpcworkqueue=128
+rpcthreads=128
+rpctimeout=220
```

Comment:
I couldn't find this one in the bitcoind codebase, can you point me to where it's documented?

Reply:
That's the problem there are not so much documentation around these options. They are out of reference config. Should I care and dig deeper? I left it on behalf of Eclair's user.

Reply:
I need to look deeper, but are you sure this is really taken into account by bitcoind?

Reply:
I see you've removed it from the last commit, all good then since I'm not sure this parameter really exists in bitcoin.